### ISSUE TITLE
Progress tracker loop bug

### ISSUE THREAD

**Issue by cam586** [AaltoPML/VAI-Lab#70]
When running `examples/xml_files/basic_operation.xml`, the first module in the loop completes, but then the progress tracker starts looking for the `Initialiser` module 

```bash
example script: src/vai_lab//examples/xml_files/basic_operation.xml

Running pipeline...
Starting for loop: "For Loop 1" ...

Traceback (most recent call last):
  File "src/vai_lab/tests/test_examples.py", line 18, in <module>
    test_examples()
  File "src/vai_lab/tests/test_examples.py", line 16, in test_examples
    core.run()
  File "src/vai_lab/Core/vai_lab_core.py", line 174, in run
    self._execute(self._xml_handler.loaded_modules)
  File "src/vai_lab/Core/vai_lab_core.py", line 149, in _execute
    getattr(self, "_execute_{}".format(specs[key]["class"]))(specs[key])
  File "src/vai_lab/Core/vai_lab_core.py", line 81, in _execute_loop
    getattr(self, "_execute_{}_loop".format(specs["type"]))(specs)
  File "src/vai_lab/Core/vai_lab_core.py", line 110, in _execute_for_loop
    self._execute(specs)
  File "src/vai_lab/Core/vai_lab_core.py", line 153, in _execute
    _tracker = self._runTracker()
  File "src/vai_lab/Core/vai_lab_core.py", line 117, in _runTracker
    return self.gui_app.launch()
  File "src/vai_lab/GUI/GUI_core.py", line 146, in launch
    frame = F(parent=container, controller=self,
  File "src/vai_lab/utils/plugins/progressTracker.py", line 65, in __init__
    self.upload()
  File "src/vai_lab/utils/plugins/progressTracker.py", line 524, in upload
    self.reset()
  File "src/vai_lab/utils/plugins/progressTracker.py", line 650, in reset
    self.add_module('Initialiser', self.width/2, self.h, ini=True)
  File "src/vai_lab/utils/plugins/progressTracker.py", line 188, in add_module
    if 'start' in self.controller.status[boxName]:
KeyError: 'Initialiser'
```

**Comment by cam586** [comment]
Being addressed in [#fab4f042](https://github.com/AaltoPML/VAI-Lab/commit/fab4f042ce8ff631e5903f6403bda1dcee02c4a9) 

Next PR will fix this bug

**Comment by cam586** [comment]
Fixed in #73
